src/index.html

```diff
@@ -1,17 +1,192 @@
+          type="checkbox"
+          name="Terms"
+          id="Terms">
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="form_field">
+          <label for="Email">
+            E-mail:
+            <input
+            type="email"
+            name="Email"
+            id="Email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required>
+          </label>
+        </div>
+
+        <div class="form_field">
+          <label for="Password">
+            Password:
+            <input
+            type="password"
+            name="Password"
+            id="Password"
+            minlength="8"
+            maxlength="32"
+            required>
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form_field">
+          Do you love cats?
+          <label for="Yes">
+            <input
+            type="radio"
+            name="Answer"
+            value="yes"
+            id="Yes">
+            Yes
+          </label>
+
+          <label for="No">
+            <input
+            type="radio"
+            name="Answer"
+            value="no"
+            id="No">
+            No
+          </label>
+        </div>
+
+        <div class="form_field">
+          What is your favorite color?
+          <input
+          type="color"
+          name="Color"
+          id="Color">
+        </div>
+
+        <div class="form_field">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="Time"
+          id="Time">
+        </div>
+
+        <div class="form_field">
+          What are your favorite brand of cars?
+          <select
+          name="brandOfCars"
+          id="brandOfCars"
+          multiple>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </div>
+
+        <div>
+          How do you rate our work?
+          <input
+          type="range"
+          name="Rate"
+          min="0"
+          max="10"
+          value="0">
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info</legend>
+        <div class="form_field">
+          Comments:
+          <textarea
+          name="Comments"
+          id="Comments"
+          cols="20"
+          rows="2">
+          </textarea>
+        </div>
+        <div class="form_field">
+          Would you recommend us?
+          <select
+            name="recommendation"
+            id="recommendation">
+            <option value="Yes">Yes</option>
+            <option value="No">No</option>
+          </select>
+        </div>
+      </fieldset>
+
+      <button type="submit">Submit</button>
+    </form>
   </body>
-</html>
+
+</html>
```

Comment:
```suggestion
</html>

```

add an empty line to the end of file